src/backend/db/categories.js

```diff
@@ -1,5 +1,13 @@
 import { v4 as uuid } from "uuid";
 
+import grouptherapy from "../../assets/home_page_images/image_gt.jpg";
+import clothing from "../../assets/home_page_images/image_lane8_model.jpg";
+import vinyl from "../../assets/home_page_images/image_hp_vinyl.png";
+import poster from "../../assets/home_page_images/poster.png";
+import cds from "../../assets/home_page_images/image_cd.png";
+import newarrivals from "../../assets/home_page_images/bottom_1.jpg";
+import trending from "../../assets/home_page_images/bottom_2.jpg";
```

Comment:
you can make file index.js in the assets folder ... from there you can export all your images ... so while importing them in categories.js you can import all images from one file